packages/bff/src/services/catalogService.ts

```diff
@@ -294,5 +308,73 @@ export function catalogServiceBuilder(
         id: updatedEservice.id,
       };
     },
+    getProducerEServices: async (
+      eserviceName: string | undefined,
+      consumersIds: string[],
+      offset: number,
+      limit: number,
+      context: WithLogger<BffAppContext>
```

Comment:
Here we could already extract `authData` and `headers` from the context

Reply:
I have deconstructed this param far all service methods in this [commit](https://github.com/pagopa/interop-be-monorepo/pull/834/commits/b716d1e90e5253de0e30a15bcb5a18760cfcabd0) 👍🏻